### ISSUE TITLE
Date Range on Field Charts

### ISSUE THREAD

**Issue by NateSchwarz** [formtools/core#403]
This isn't technically an issue, but a question about functionality. I'm trying to find a way to have a date range (ex: past 30 days) for the field charts in the data visualization module. I see there is that functionality on the activity charts, so maybe I'm just missing something. 

The goal is to have a pie chart, for example, with data from a field showing for only the past thirty days and have that be dynamic so I don't have to keep setting the time frame. Is there a way to do this in the system?

**Comment by benkeen** [comment]
Hey @NateSchwarz, 

Strange, I wonder why I never added a date range to the Field Chart option. It does seem like a pretty simple thing to add! Let me take a look at it tonight. Honestly, I need a break and this seems like it might be nice and small. ;-) 

Keep you posted.

**Comment by benkeen** [comment]
Yeah this looks fine. Unless I hit any hiccups I'll try to get a new version out tomorrow.

**Comment by NateSchwarz** [comment]
Hey @benkeen,

I greatly appreciate it! I love the tool, and I guess I just keep finding new ways I want to use it!

Thanks so much.

**Comment by benib596** [comment]
Hi Ben,

Its nice the data visualization with data range.  One small suggestion, it is best the view option also in the appearance tab so that we can easily switch with different view (users) and find out the output.  Also is it possible to put select multiple view option to compare the selected two or more views.
